Cargo.toml

```
@@ -20,24 +20,9 @@ license = "Apache-2.0"
 repository = "https://github.com/delta-io/delta-kernel-rs"
 readme = "README.md"
 rust-version = "1.80"
-version = "0.6.1"
+version = "0.7.0"
```

I usually do the updates to version all at once during a release

```suggestion
version = "0.6.1"
```